client/src/components/game/static/GameBoardTop.vue

```diff
@@ -38,39 +32,28 @@
           </div>
         </div>
       </div>
-
-      <!-- REFACTOR :: MarsLogContainer -->
-      <div class="marslog col-4">
-        <div class="wrapper">
-          <NewMarsLog />
-        </div>
-      </div>
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import { Vue, Component } from 'vue-property-decorator';
-import StatusBar from '@port-of-mars/client/components/gamedashboard/global/StatusBar.vue';
-import Player from '@port-of-mars/client/components/gamedashboard/top/Player.vue';
-import OtherPlayers from '@port-of-mars/client/components/gamedashboard/top/OtherPlayers.vue';
-import GameInformation from '@port-of-mars/client/components/gamedashboard/top/GameInformation.vue';
-import PhaseInstructions from '@port-of-mars/client/components/gamedashboard/top/PhaseInstructions.vue';
-import NewMarsLog from '@port-of-mars/client/components/gamedashboard/top/NewMarsLog.vue';
+import Player from './playerPanel/Player.vue';
```

Comment:
same concerns here as elsewhere. How about merging `informationPanel` and `playerPanel` into `panels` with all 4 of those `.vue` files living in the same directory. You'll need to do the same with their referenced `.scss` files